pkg/resourceinterpreter/default/thirdparty/resourcecustomizations/batch.volcano.sh/v1alpha1/Job/customizations.yaml

```diff
@@ -0,0 +1,251 @@
+apiVersion: config.karmada.io/v1alpha1
+kind: ResourceInterpreterCustomization
+metadata:
+  name: declarative-configuration-job
+spec:
+  target:
+    apiVersion: batch.volcano.sh/v1alpha1
+    kind: Job
+  customizations:
+    replicaRevision:
+      luaScript: >
+        function ReviseReplica(obj, replica)
+          obj.spec.minAvailable = replica
+          return obj
+        end
```

Comment:
I guess we shouldn't implement the `replicaRevision`, as currently we don't support `divide` (Volcano) Job yet.

```suggestion
```